azerothcore/azerothcore-wotlk
# feat(Core/Player): Allow stay in group when logged out

## Changes Proposed:
-  Add a config option that allows a player to stay in their group when they log out

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Set the config option to false
- Logged two characters into the world
- Logged one of them out to the character selection screen
- Logged it back in and saw the group still together
- Logged both characters out and back in again and checked the group again
- Repeated above steps with the option set to true


## How to Test the Changes:

Check the below with both true and false set

1. Log at least two characters into the world
2. Join them together in a group
3. Log one, or all, of them out
4. Enter the world again and check the party

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
